akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -311,7 +311,16 @@ private[stream] class VirtualSubscriber[T](val owner: VirtualPublisher[T]) exten
  */
 private[stream] class VirtualPublisher[T]() extends Publisher[T] {
   @volatile var realPublisher: Publisher[T] = null
-  override def subscribe(s: Subscriber[_ >: T]): Unit = realPublisher.subscribe(s)
+  override def subscribe(s: Subscriber[_ >: T]): Unit = {
+    val sub = realPublisher.subscribe(s)
+    realPublisher match {
+      case _: SupportingMultipleSubscribers   ⇒
```

Comment:
Just to prevent misunderstandings: what I want eventually is that the behavior is consistently “single-subscriber”, i.e. the removal of that feature is completely fine. I just don’t want to spend the effort right now of figuring out why those unrelated tests fail as a result (with the exception of one case that tests for a feature that shouldn’t be there).